config/default.yaml

```diff
@@ -36,4 +36,7 @@ redis:
     # Password to connect to redis cluster
     # password: null
     # Host of redis cluster
-    port: 6379
+    port: "6379"
```

Comment:
node-resque should be okay, changed this since in k8s deployment yaml it will need a quote for this. But in the config yaml, I think this is valid. Changed it back! thanks Joel